### PR TITLE
docs: adding link to duckdb docs

### PR DESCRIPTION
**Pull request by edublancas**
Hi! I added a link to DuckDB's docs to the README (which uses JupySQL 😁). Let me know what you think!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,8 +58,8 @@ assert frank.name == "Frank"
 
 ## Usage in IPython/Jupyter
 
-With IPython-SQL and DuckDB-Engine you can query DuckDB natively in your notebook!
-Alex Monahan has a great demo of this on [his blog](https://alex-monahan.github.io/2021/08/22/Python_and_SQL_Better_Together.html#an-example-workflow-with-duckdb)
+With IPython-SQL and DuckDB-Engine you can query DuckDB natively in your notebook! Check out [DuckDB's documentation](https://duckdb.org/docs/guides/python/jupyter) or
+Alex Monahan great demo of this on [his blog](https://alex-monahan.github.io/2021/08/22/Python_and_SQL_Better_Together.html#an-example-workflow-with-duckdb).
```
Comment (Mause) [inline README.md:62]:
```suggestion
Alex Monahan's great demo of this on [his blog](https://alex-monahan.github.io/2021/08/22/Python_and_SQL_Better_Together.html#an-example-workflow-with-duckdb).
```

**Review by Mause — CHANGES_REQUESTED** [review]

**Comment by Mause** [comment]
@edublancas are you able to fix up the commit message and the grammar? Happy to merge once that's done

**Comment by edublancas** [comment]
looks like the commit name is good now :)

**Comment by Mause** [comment]
Thanks!

**Comment by edublancas** [comment]
woohoo 🎉

**Pull request merged**